cmd/containerboot/main.go

```diff
@@ -52,6 +52,12 @@
 //     ${TS_CERT_DOMAIN}, it will be replaced with the value of the available FQDN.
 //     It cannot be used in conjunction with TS_DEST_IP. The file is watched for changes,
 //     and will be re-applied when it changes.
+//   - TS_HEALTHCHECK_ADDR_PORT: if specified, an HTTP health endpoint will be
+//     served at /healthz at the provided address, which should be in form [<address>]:<port>.
+//     If not set, no health check will be run. If set to :<port>, addr will default to 0.0.0.0
```

Comment:
do we need to specify the addr here in the comment?

Reply:
> do we need to specify the addr here in the comment?

Could you clarify- do you mean the comment should have less info or more?

Reply:
I'm going to merge this now, but we can improve the comment.